regparser/index/dependency.py

```diff
@@ -110,7 +110,8 @@ def validate_for(self, entry):
         logger.debug("Validating dependencies for %r", key)
         for dependency in self.dependencies(key):
             if self.node(dependency).get('stale'):
-                raise Missing(key, self.node(dependency)['stale'])
+                pass
+                # raise Missing(key, self.node(dependency)['stale'])
```

Comment:
This line is definitely needed. These exceptions trigger other commands to run, like `fetch_annual_edition`, which does the leg work of downloading and caching the missing file.